http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -150,7 +150,7 @@ class GceInterpreter[F[_]](
               )
             )
             isFormatted <- persistentDisk.formattedBy match {
-              case Some(FormattedBy.Galaxy) | Some(FormattedBy.Custom) =>
+              case Some(FormattedBy.Galaxy) | Some(FormattedBy.Custom) | Some(FormattedBy.Cromwell) =>
```

Comment:
I wish there was some way that we could avoid having to explicitly list the different app types to make it easier to add new ones. Could we have a property about App vs. Runtime so we could iterate through the Apps?